demos/llamastack/Llama_Stack_NVIDIA_E2E_Flow.ipynb

```diff
@@ -0,0 +1,2924 @@
+      "Job status: completed after 335.8395550251007 seconds.\n"
+     ]
+    }
+   ],
+   "source": [
+    "job = wait_customization_job(job_id=job_id, polling_interval=30, timeout=3600)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 60,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Job cust-DHndZjk1zmqeJDf7FMJuoe status: completed\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(f\"Job {job_id} status: {job_status}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Checking if NIM has loaded customized model nvidia-e2e-tutorial/test-messages-model@v1763646228.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 10.596751928329468 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 21.086602926254272 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 31.566166877746582 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 42.22009587287903 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 52.86898708343506 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 63.35778498649597 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 73.84983396530151 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 84.33940100669861 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 94.82941198348999 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 105.3087830543518 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 not available after 115.79959392547607 seconds.\n",
+      "Model nvidia-e2e-tutorial/test-messages-model@v1763646228 available after 126.27090287208557 seconds.\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Check that the customized model has been picked up by NIM;\n",
+    "# We allow up to 5 minutes for the LoRA adapter to be loaded\n",
+    "wait_nim_loads_customized_model(model_id=customized_chat_model_dir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 62,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "✅ Inference response: -----------\n",
+      "A) blue\n",
+      "B) purple\n",
+      "C) white\n",
+      "D) yellow\n",
+      "\n",
+      "The best answer\n"
+     ]
+    }
+   ],
+   "source": [
+    "import requests\n",
+    "\n",
+    "response = requests.post(\n",
+    "    f\"{NIM_URL}/v1/completions\",\n",
```

Comment:
When you have time, please investigate how to register models via the llamastack server, and execute completions with the new models via the llamastack server.